lib/hanami/cli/commands/app/generate/repo.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "dry/inflector"
+require "dry/files"
+require "shellwords"
+require_relative "../../../naming"
+require_relative "../../../errors"
+
+module Hanami
+  module CLI
+    module Commands
+      module App
+        module Generate
+          # @since 2.2.0
+          # @api private
+          class Repo < Command
+            argument :name, required: true, desc: "Repo name"
+            option :slice, required: false, desc: "Slice name"
+
+            example [
+              %(books               (MyApp::Repos::BooksRepo)),
+              %(books/drafts_repo   (MyApp::Repos::Books::DraftsRepo)),
+              %(books --slice=admin (Admin::Repos::BooksRepo)),
+            ]
+
+            # @since 2.2.0
+            # @api private
+            def generator_class
+              Generators::App::Repo
+            end
+
+            # @since 2.2.0
+            # @api private
+            def call(name:, slice: nil, **opts)
+              normalized_name = if name.end_with?("_repo")
+                                  name
+                                else
+                                  "#{inflector.pluralize(name)}_repo"
+                                end
+              slice = inflector.underscore(Shellwords.shellescape(slice)) if slice
```

Comment:
TBH, I don't think we need to shellescape this slice arg at all. We're checking for the presence of the slice dir via our generators anyway, so that's validation enough for the slice name.

In https://github.com/hanami/cli/pull/201 I removed the shellescaping, and I reckon we could just do the same here too.